src/IO.jl

```diff
@@ -49,56 +51,36 @@ function set_active_demand(alldata::Array{Dict{Any,Any}}, demand::Array{Float64,
 end
 
 "Organize Parameters"
-function set_param(;stages::Int = 1,model_constructor_grid = DCPPowerModel, post_method = PowerModels.post_opf,solver = ClpSolver(),setting = Dict("output" => Dict("branch_flows" => true)))
+function set_param(;stages::Int = 1,
+                    model_constructor_grid = DCPPowerModel, 
+                    post_method = PowerModels.post_opf,optimizer = Clp.Optimizer,
+                    setting = Dict("output" => Dict("branch_flows" => true,"duals" => true)),
+                    verbose = false)
     params = Dict()
     params["stages"] = stages
     params["model_constructor_grid"] = model_constructor_grid
     params["post_method"] = post_method
-    params["solver"] = solver
+    params["optimizer"] = optimizer
+    params["verbose"] = verbose
     params["setting"] = setting
     return params
 end
 
-"Build Solution single simulation"
-function build_solution_single_simulation(m::SDDPModel;solution = Dict())
-    # add results    
-    stages = size(m.stages,1) # count number of stages
-
-    solution["solution"]= Dict()
-    solution["solution"]=Array{Dict}(stages)
-    for s = 1:stages
-        i = solution["markov"][s]       
-        built_sol = PowerModels.build_solution(m.stages[s].subproblems[i].ext[:pm],:Optimal,0.0,
-        solution_builder = PowerModels.get_solution)
-        solution["solution"][s] = built_sol["solution"]
-        solution["solution"][s]["objective"] = built_sol["objective"]
-        solution["solution"][s]["objective_lb"] = built_sol["objective_lb"]
-        solution["solution"][s]["reservoirs"] = Dict()
-        for r =1:m.ext[:alldata][1]["hydro"]["nHyd"]
-            solution["solution"][s]["reservoirs"]["$r"] = Dict()
-            solution["solution"][s]["reservoirs"]["$r"]["spill"] = getvalue(m.stages[s].subproblems[i][:spill])[r]
-            solution["solution"][s]["reservoirs"]["$r"]["outflow"] = getvalue(m.stages[s].subproblems[i][:outflow])[r]
-            solution["solution"][s]["reservoirs"]["$r"]["volume"] = getvalue(m.stages[s].subproblems[i][:reservoir])[r]
-        end        
-    end
-    return solution
-end
-
 """Quantile Scenarios"""
 function quantile_scen(scen::Array{Float64,2},quant::Float64)
-    return [quantile(scen[i, :], quant) for i = 1:size(scen, 1)]
+    return [Statistics.quantile(scen[i, :], quant) for i = 1:size(scen, 1)]
 end
 
 """Quantile Scenarios"""
 function quantile_scen(scen::Array{Float64,2},quants::Array{Float64};output_dict::Bool=false)
     if output_dict
         output = Dict()
         for quant in quants 
-            output["$(quant*100)%"] = [quantile(scen[i, :], quant) for i = 1:size(scen, 1)]
+            output["$(quant*100)%"] = [Statistics.quantile(scen[i, :], quant) for i = 1:size(scen, 1)]
         end
         return output
     else
-        return [quantile(scen[i, :], quant) for i = 1:size(scen, 1),quant in quants]
+        return [Statistics.quantile(scen[i, :], quant) for i = 1:size(scen, 1),quant in quants]
```

Comment:
In order to use it I would have to either rewrite the above function, because it only creates a vector, or concatenate all vectors, which could be less efficient I believe. The original idea behind having different functions was to facilitate me to name each quantile when plotting.